app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+            # Include default devise modules.
```

Comment:
Why is this here? what is devise? we are installing rubocop on this PR not devise :p